src/schemas/json/stylelintrc.json

```diff
@@ -1175,6 +1204,7 @@
                           "enum": [
                             "between-comments",
                             "after-comment",
+                            "stylelint-command",
```

Comment:
https://github.com/stylelint/stylelint/blob/7aa2282ec270276bbd52fad48d19518972a2ced5/lib/rules/comment-empty-line-before/README.md?plain=1#L131

similar to #350